### PR TITLE
codeexecutor: omit empty file content

### PR DESCRIPTION
**Pull request by WineChord**
## What
- Add `omitempty` to `codeexecutor.File.Content` JSON tag.

## Why
- Avoid emitting an empty `content` field when only `size_bytes` is known.
  This looks semantically confusing to LLMs.

## Tests
- `go test ./...`

## Summary by Sourcery

使文件的 JSON 序列化在保留其他元数据的同时省略空的 `content` 值。

Bug Fixes:
- 当只知道文件大小时，避免在 JSON 中生成空的 `content` 字段。

Tests:
- 添加测试，以验证当内容为空时文件的 `content` 会从 JSON 中省略，而在非空时会被包含。

<details>
<summary>Original summary in English</summary>

## Summary by Sourcery

Make file JSON serialization omit empty content values while preserving other metadata.

Bug Fixes:
- Avoid emitting an empty JSON content field for files when only the size is known.

Tests:
- Add tests verifying that file content is omitted from JSON when empty and included when non-empty.

</details>

### CODE REVIEW

File: codeexecutor/codeexecutor.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func (r CodeExecutionResult) String() string {
 // File represents a file generated during code execution.
 type File struct {
 	Name      string `json:"name"`
-	Content   string `json:"content"`
+	Content   string `json:"content,omitempty"`
```
Comment (Flash-LHR) [inline codeexecutor/codeexecutor.go:63]: 文档约定 `content` 为空字符串时仍返回字段，加 `omitempty` 会把字段省略导致契约不一致。建议去掉 `omitempty` 或同步更新协议与文档。

Comment (WineChord) [reply codeexecutor/codeexecutor.go:63]: fixed

File: docs/mkdocs/zh/skill.md [hunk 2]
```diff
@@ -350,8 +350,8 @@ agent := llmagent.New(
 - `output_files`：文件列表（`name`、`ref`、`content`、`mime_type`、
   `size_bytes`、`truncated`）
   - `ref` 是稳定的 `workspace://<name>` 引用，可传给其它工具使用
-  - 非文本文件的 `content` 始终为空。
```
Comment (WineChord) [inline docs/mkdocs/zh/skill.md:353]: Image

Image

**Review by sandyskies — APPROVED** [review]
LGTM
